README.md

```diff
@@ -56,43 +56,43 @@ Note: If the module block has explicit version specified, then the version must
 For example, in the below config the module has version 0.19.0 as shown below:
 ```hcl
 module "clumio_aws_connection_module" {
-providers = {
-aws    = aws
-clumio = clumio
-}
-source                = "../../"
-version               = "0.19.0"
-clumio_token          = clumio_aws_connection.test_conn.token
-role_external_id      = clumio_aws_connection.test_conn.role_external_id
-aws_region            = clumio_aws_connection.test_conn.aws_region
-aws_account_id        = data.aws_caller_identity.current.account_id
-clumio_aws_account_id = clumio_aws_connection.test_conn.clumio_aws_account_id
-is_ebs_enabled        = true
-is_rds_enabled        = true
-is_ec2_mssql_enabled  = true
-is_s3_enabled         = true
-is_dynamodb_enabled   = true
+  providers = {
+  aws    = aws
+  clumio = clumio
+  }
+  source                = "../../"
+  version               = "0.26.0"
```

Comment:
should this be 0.27.0

Reply:
No. This is to show how to upgrade from 0.26.0 to 0.27.0